js/google_ima.js

```diff
@@ -279,6 +279,12 @@ require("../html5-common/js/utils/utils.js");
           this.useInsecureVpaidMode = metadata.vpaidMode === "insecure";
         }
 
+        this.disableFlashAds = false;
```

Comment:
Should we also define this and set to false in the _resetVars function? The other page level settings are treated like this.